setup.py

```diff
@@ -41,7 +41,7 @@ def get_package_data(package, base_dir):
 
 kwargs = {
     "name": "grouper",
-    "version": __version__,  # type: ignore  # noqa: F821
+    "version": __version__,  # type: ignore[name-defined]  # noqa: F821
```

Comment:
Yeah, I left this in because I'm hoping to re-enable F821 soon. A diff to support the new mypy ignore syntax ha been merged, so as soon as they cut a new release, we can update to that and turn that error back on.